.teamcity/_self/projects/WPComPlugins.kt

```diff
@@ -184,3 +188,38 @@ private object InlineHelp : WPComPluginBuild(
 	docsLink = "TODO",
 	withPRNotify = "false",
 )
+
+private object GutenbergUploadSourceMapsToSentry: BuildType() {
```

Comment:
Nit: we could (probably?) change this syntax to:

```
... : BuildType({
  name = ""
  params {
  ...
})
```

Basically removing the need to call `init` directly

Reply:
Thanks, took note and will look in a follow-up :)